docs/static/setup.sh

```diff
@@ -121,6 +121,7 @@ select opt in "${options[@]}" "Quit"; do
     19 ) shield_title="Eek!" shield="eek"; split="n" break;;
     20 ) shield_title="BFO-9000" shield="bfo9000"; split="y"; break;;
     21 ) shield_title="Helix" shield="helix"; split="y"; break;;
+    21 ) shield_title="Dactyl Manuform 4x5" shield="dactyl_manuform_4x5"; split="y"; break;;
```

Comment:
```suggestion
    22 ) shield_title="Dactyl Manuform 4x5" shield="dactyl_manuform_4x5"; split="y"; break;;
```

Reply:
Aha, good eyes, thank you! I've committed your suggestion 😊